styles.css

```diff
@@ -110,12 +117,314 @@ section {
 }
 
 .second-section p:nth-child(3) {
+  color: #344563;
   margin-top: 12px;
-  width: 90%;
 }
 
 .social-container li {
-  margin-right: 19px;
-  font-size: 18px;
+  margin-right: 20px;
+  font-size: 25px;
+  width: 20px;
+  color: #505f79;
+}
+
+.projects {
+  display: grid;
+  grid-template-columns: 100%;
+  grid-column-gap: 50px;
+  width: 100%;
+  margin: auto;
+}
+
+.project {
+  margin: 80px 24px 0;
+  border: 1px solid #dfe1e6;
+  display: flex;
+  flex-direction: column;
+  padding-bottom: 27px;
+  border-radius: 16px;
+  background-color: white;
+  font-family: "Poppins", sans-serif;
+}
+
+.project > * {
+  margin: 0 16px;
+}
+
+.project img {
+  margin-top: 16px;
+}
+
+.project-image {
+  margin: auto;
+}
+
+.project-name {
+  font-size: 32px;
+  font-weight: bolder;
+  margin-top: 10px;
+  margin-bottom: 20px;
+  width: 70%;
+  color: #172b4d;
+}
+
+.project-year {
+  margin-bottom: 20px;
+  display: flex;
+  justify-content: space-between;
+  width: 70%;
+}
+
+.canopy {
+  font-size: 13px;
+  font-weight: bolder;
+  color: #344563;
+}
+
+.role {
+  font-size: 13px;
+  color: #7a869a;
+  font-weight: bolder;
+}
+
+.dot {
+  background-color: #c1c7d0;
+  width: 5px;
+  height: 5px;
+  border-radius: 100%;
+  position: relative;
+  bottom: -7px;
+}
+
+.project-description {
+  line-height: 25px;
+  color: #344563;
+}
+
+.project-technologies {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 20px 0 25px;
+}
+
+.project-technologies > * {
+  margin-right: 10px;
+  border-radius: 8px;
+  background-color: #ebebff;
+  padding: 5px 10px;
+  color: #6070ff;
+  font-size: 12px;
+}
+
+.project-button {
+  width: 130px;
+  height: 48px;
+  background-color: white;
+  border: 2px solid #6070ff;
+  border-radius: 8px;
+  font-size: 17px;
+  color: #396df2;
+  font-weight: 500;
+}
+
+.project-button:hover {
+  background-color: #6070ff;
+  border: 2px solid #6070ff;
+  border-radius: 8px;
+  font-size: 17px;
+  color: #fff;
+  font-weight: 500;
+}
+
```

Comment:
- [ ] The button (e.g., "See project") is a link a or button tag, and follow the button styling: Active, Hover and Pressed (see design), hover is not working please fix it.
- [ ] The cursor property specifies the mouse cursor to be displayed when pointing over an element. please use cursor pointer to make your button look nice, refer on [this](https://www.w3schools.com/cssref/pr_class_cursor.asp) for more details

Reply:
I have fixed this now. I am so happy about what you pointed out.

Reply:
I used a button tag for the buttons. I tried using a tag to wrap it but it gave me an error. 

I also tested the hover effect on the buttons and it works on the hosted page on Github. Below are some screenshots to drive home my point.

Thank
![Screenshot from 2021-07-09 20-46-06](https://user-images.githubusercontent.com/45668340/125129356-3c692600-e0f7-11eb-8adf-e44c23b7e2ff.png)
![Screenshot from 2021-07-09 20-47-21](https://user-images.githubusercontent.com/45668340/125129360-3d9a5300-e0f7-11eb-9428-5ee54ef080f9.png)
 you.